node_modules/oae-core/about/css/about.css

```diff
@@ -0,0 +1,62 @@
+
+#about-description .about-description-empty,
+#about-description .about-description-empty a {
+    color: #888;
```

Comment:
I'd prefer not to hardcode this color here, but it's probably not worth adding a skin value for it, so no action needed